### ISSUE TITLE
Provide minimal quickstart .nengobones.yml file

### ISSUE THREAD

**Issue by jgosmann** [nengo/nengo-bones#24]
It would be useful to have a minimal working .nengobones.yml file to get basic things running quickly and get a *quick* overview of the format (which I think the current documentation does not do a good job of). Maybe [this file](https://github.com/nengo/nengo-bones/blob/master/.nengobones.yml) is already pretty minimal, but then it is quite a lot of boilerplate (and it definitely has some unnecessary environment variable defined).

**Comment by drasmuss** [comment]
Yes this is something we will add.  For now though, the best place to start is probably https://github.com/nengo/nengo-dl/blob/master/.nengobones.yml, as the nengo-dl `.nengobones.yml` has the simplest configuration.
